### ISSUE TITLE
Writing a letter overlaps the menu 

### ISSUE THREAD

**Issue by Cairothefox** [urnotnick/impulse-issues#201]
**Describe the bug**
When you are go to write a letter in the F4 quick actions menu. the writing letter prompt overlaps the Menu

**To reproduce**
Steps to reproduce the behaviour:
1. Go to Quick actions in F4
2. Click on Write a letter

**Expected behaviour**
The Write a letter prompt should not overlap the menu.

**Screenshots**
https://i.imgur.com/9fVcBNB.png

**impulse version (press F1 in the bottom right):** 1.12.0

**Lua error**
If applicable, post the error here in full.

**Additional context**

**Comment by justinbrick** [comment]
An interesting issue. Is your monitor a classic 16:9 display?

**Comment by Cairothefox** [comment]
Yes mine is.Sent from my iPhoneOn 23 Feb 2023, at 21:21, Justin ***@***.***> wrote:﻿
An interesting issue. Is your monitor a classic 16:9 display?

**Comment by justinbrick** [comment]
What resolution is it?
